google-cloud-clients/google-cloud-bigtable/README.md

```diff
@@ -45,7 +45,7 @@ compile 'com.google.cloud:google-cloud-bigtable:1.5.0'
 ```
 If you are using SBT, add this to your dependencies
 ```Scala
-libraryDependencies += "com.google.cloud" % "google-cloud-bigtable" % "1.5.0"
+libraryDependencies += "com.google.cloud" % "google-cloud-bigtable" % "1.6.0"
```

Comment:
Update bigtable version for groovy as well.

Reply:
done